apps/desktop/src/lib/gitHost/github/githubPrService.ts

```diff
@@ -1,34 +1,39 @@
 import { GitHubPrMonitor } from './githubPrMonitor';
 import { DEFAULT_HEADERS } from './headers';
 import { ghResponseToInstance, parseGitHubDetailedPullRequest } from './types';
+import { showError } from '$lib/notifications/toasts';
 import { sleep } from '$lib/utils/sleep';
 import posthog from 'posthog-js';
-import { writable } from 'svelte/store';
+import { get, writable } from 'svelte/store';
+import type { Persisted } from '$lib/persisted/persisted';
 import type { RepoInfo } from '$lib/url/gitUrl';
 import type { GitHostPrService } from '../interface/gitHostPrService';
 import type { DetailedPullRequest, MergeMethod, PullRequest } from '../interface/types';
 import type { Octokit } from '@octokit/rest';
 
+const DEFAULT_PULL_REQUEST_TEMPLATE_PATH = '.github/PULL_REQUEST_TEMPLATE.md';
```

Comment:
@ndom91 I just realized the gh docs say this filename should be lowercase. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

<img width="533" alt="image" src="https://github.com/user-attachments/assets/4205c789-5dbd-42f4-bc23-0e6b0378e3e4">